pandas/core/frame.py

```diff
@@ -826,6 +826,11 @@ def iteritems(self):
             for i, k in enumerate(self.columns):
                 yield k, self._ixs(i, axis=1)
 
+    @Appender(items.__doc__)
+    def iteritems(self):
+        for key, value in self.items():
```

Comment:
you don't want to return self.items() here?

Reply:
Yeah, I've changed that.

The issue was that items is a generator, while iteritems is a function, that returns a generator. So flake8 requires the items doc string to use "Yields" and the iteritems doc string to use "Returns".

This is a bit silly in either case, and it feels off to do stuff just to satisfy a linter.